lib/fluent/plugin_helper/cert_option.rb

```diff
@@ -171,9 +171,12 @@ def cert_option_generate_server_pair_self_signed(generate_opts)
 
       def cert_option_certificates_from_file(path)
         data = File.read(path)
-        pattern = Regexp.compile('-+BEGIN CERTIFICATE-+\n(?:[^-]*\n)+-+END CERTIFICATE-+\n?', Regexp::MULTILINE)
+        pattern = Regexp.compile('-+BEGIN CERTIFICATE-+\r?\n(?:[^-]*\r?\n)+-+END CERTIFICATE-+\r?\n?', Regexp::MULTILINE)
         list = []
         data.scan(pattern){|match| list << OpenSSL::X509::Certificate.new(match) }
+        if list.length() == 0
```

Comment:
`()` isn't necessary 
```suggestion
        if list.length == 0
```